gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/voltage/json/FailureVoltageCheckResult.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ *  This Source Code Form is subject to the terms of the Mozilla Public
+ *  License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.swe.runner.app.voltage.json;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * @author Marc Schwitzguébel {@literal <marc.schwitzguebel at rte-france.com>}
+ */
+public class FailureVoltageCheckResult {
+
+    private static final String IS_SECURE = "FAILURE";
```

Comment:
Why use a constant here? I know magic-strings are bad code, but is this constant really useful?

Maybe the question is not really "why use a constant" but "why define a method called `getIsSecure` if there is only one possible return value?". Shouldn't we extend a class or implement an interface?